### ISSUE TITLE
[Feature request] Add Belarusian pre-recorded voice guidance

### ISSUE THREAD

**Issue by annasoloviova** [osmandapp/Osmand#3649]
User requests Belarusian pre-recorded voice guidance.

According to them, there are  reasons to use recorded voice: 
1) Native TTS engine on Android device doesn't support Belarusian. 
2) Third-party TTS engines for Belarusian doesn't work well: one of them sounds ugly and requires connection to TTS server, second one doesn't support digits.

**Comment by xmd5a2** [comment]
Observation: current google tts does not support belarussian language

**Comment by alex73** [comment]
OsmAnd had Belarusian pre-recorded voice some times ago. But this voice was removed.
What happen ? Should we add some messages ?
Is there any instruction how to add prerecorded voice ?

**Comment by vshcherb** [comment]
https://github.com/osmandapp/OsmAnd-resources/tree/master/voice/be it is still possible to pack and create voice (voice files are there) but we don't plan to publish it through the application.

**Comment by vshcherb** [comment]
Adding a package is just copying the configuration & voice files to the folder

**Comment by alex73** [comment]
Victor, why do you not plan to publish it ? Package should be improved for that, or you are planning to leave only TTS voices and drop prerecorded voices ?

**Comment by vshcherb** [comment]
Yes we plan to keep mostly TTS languages cayse it is easier to support them. Also many original contributors didn't update the missing voice languages, so it makes hard to implement any new feature

**Comment by DmitryAlexei** [comment]
Done
Belarusian language is available as TTS and recorded voice.
